samples/jwt-server/jwt-server.yaml

```diff
@@ -25,9 +25,70 @@ spec:
   - name: http
     port: 8000
     targetPort: 8000
+  - name: https
+    port: 8443
+    targetPort: 8443
   selector:
     app: jwt-server
 ---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: jwt-cert-key-secret
+# command to generate certificate
+# https://github.com/istio/istio/blob/master/samples/jwt-server/testdata/README.MD
+stringData: 
+  example.crt: |
```

Comment:
Let's use the `server.crt` and `server.key` to match with the name used in the README.

Reply:
sure!!